exposee-app/server.js

```diff
@@ -36,51 +37,70 @@ app.use(
     }
   })
 );
+ 
 sessionStore.sync();
 
 app.use(userRoutes);
 
-// Route to get all posts, with associated users
-app.get('/posts', async (req, res) => {
+
+app.get('/videos', async (req, res) => {
+
   try {
-    const posts = await Post.findAll({
-      include: [{ model: User, as: 'user' }],
-      order: [['createdAt', 'DESC']]
-    });
-    res.json(posts);
-  } catch (err) {
-    res.status(500).json({ message: err.message });
+    const videos = await Video.findAll();
+    console.log(videos)
+  res.json(videos)}
+   catch (err) {
+  console.error(err);
+  res.status(500).json({ message: err.message });
+   }}
+)
+app.post('/videos', async (req, res) => {
+  try {
+    const {title ,url} = req.body;
+    const videos = await Video.create({title, url})
+    res.status(201).json(videos);
+  } catch(error){
+  console.error(error);
+  res.status(500).json({ message: error.message });
   }
 });
 
-// Route to create a new post
-app.post('/posts', async (req, res) => {
-  try {
-    // Check if user is logged in
-    if (!req.session.user) {
-      return res.status(401).json({ error: 'Unauthorized' });
-    }
+app.get('/videos/:id', async(req,res) => {
 
-    // Retrieve the current user from the session
-    const currentUser = req.session.user;
+  try {
+    const videoId = req.params.id;
+    const videos = await Video.findByPk(videoId)
```

Comment:
so when you use findByPk, it should return 1 video, thus calling this `const video` makes more sense